tests/CMakeLists.txt

```diff
@@ -109,14 +109,3 @@ add_test(
   NAME    hash-target
   COMMAND hash-target-tests)
 
-set(enabled_tests
-    coretests
-    difficulty
-    hash
-    performance_tests
-    core_proxy
-    fuzz
-    unit_tests)
-
-add_custom_target(tests DEPENDS enabled_tests)
-set_property(TARGET tests PROPERTY FOLDER "${folder}")
```

Comment:
This is still needed. The only change would be to replace `...DEPENDS enabled_tests...` with `DEPENDS coretests difficulty .... unit_tests)`. But, even this needs testing, I'm not sure if `enabled_tests` has some special meaning (not likely).

Reply:
I just undid this change. I am not 100 % sure what tests it is dependent on at this point.  It was an unrelated change anyhow and should be in a separate commit.